packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -378,6 +425,83 @@ function commitHookEffectListMount(tag: number, finishedWork: Fiber) {
   }
 }
 
+function invokePassiveEffectCreate(effect: HookEffect): void {
+  const create = effect.create;
+  effect.destroy = create();
+}
+
+// TODO: Remove this duplication.
+function commitHookEffectListMount2(fiber: Fiber): void {
```

Comment:
Same question about `commitHookEffectListMount2`. Why is this showing up as though it's new?